presto-native-execution/presto_cpp/main/tests/PrestoQueryRunner.cpp

```diff
@@ -100,7 +100,9 @@ class ServerResponse {
     const auto& error = response_["error"];
 
     VELOX_FAIL(
```

Comment:
We should delete this file. The code was moved to Velox.

Reply:
Can we do this in a follow-up PR?

Reply:
PR here https://github.com/prestodb/presto/pull/21822